src/peer_connection/mod.rs

```diff
@@ -1325,19 +1322,20 @@ impl RTCPeerConnection {
                                 };
```

Comment:
the `if let Some(t) = ...` unpacking here seems a bit strange when we just repack it `Some(t)`. Maybe use `or_else`?

Reply:
I've been thinking the same thing, but it's not really in scope for this PR to change it. There's room for a PR that tweaks this pattern across the codebase, personally I think match and an early return to keep nesting levels in check is a good choice